### ISSUE TITLE
[Question] Passing Rails controller actions as props

### ISSUE THREAD

**Issue by LauraAubin** [reactjs/react-rails#933]
### Why would this be useful?

Imagine if you have rendered a card with React that contains a title, some content text, and a button. Lets say that button opened up a Rails form, how would you pass the Rails action to the component?

Likewise, would it be possible to send modified data/props/state values back up to a helper method to be stored in the Rails DB?

**Comment by BookOfGreg** [comment]
I'm sorry but I'm not quite sure what you're asking. Could you try rephrase it to help me understand? Perhaps a scenario would help.

**Comment by LauraAubin** [comment]
Say you have `home.html.erb` and you render a React component called `Frame`. Now inside the React component you can style the page and add content, but say inside this page you also want a button called `Submit form` to open another rails view called `submit.html.erb`. How can you call that action inside of the React component?

**Comment by LauraAubin** [comment]
Another use case would be if you wanted to store the value of the component's state in a variable used in a Rails model. 

In general, I'm curious if there is a way to get data to come back up from the React component (like a callback)

**Comment by BookOfGreg** [comment]
Context is not passed between ruby and js environments through anything other than props. If there's anything you want in React land then it must be passed in initially. If there's anything you need client-side at run time, you need to fetch it via ajax and therefore have an endpoint that will serve up what you are after.
